artemis-server/src/main/java/org/apache/activemq/artemis/core/config/impl/ConfigurationImpl.java

```diff
@@ -2848,6 +2849,16 @@ public Configuration setSuppressSessionNotifications(boolean suppressSessionNoti
       return this;
    }
 
+   @Override
+   public String getStatus() {
+      return status;
+   }
+
+   @Override
+   public void setStatus(String status) {
```

Comment:
can you add a javadoc here:

Some free string that will define the server's status. a Generic mechanism to be set through the configuration?